projects/bp-gallery/src/components/top-and-bottom-bar/NavigationBar.tsx

```diff
@@ -71,7 +71,7 @@ const NavigationBar = ({ isMobile }: { isMobile?: boolean }) => {
 
   return (
     <>
-      <div className='nav-bar'>
+      <div className='nav-bar shrink-0'>
```

Comment:
This change is from 3baa7c20fe5871908a8e75a950603181dfd830cc, but you reverted most of the changes in later commits. Did you forget to remove this one or does it have a purpose?